content/r/r.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: 'R'
+Description: 'R is a popular open-source interpreted programming language used for statistical computing and graphics.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/r'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+R is a popular open-source interpreted programming language used for statistical computing and graphics.
+It was created by Ross Ihaka and Robert Gentleman and version 1.0 was released in February 2000.
```

Comment:
```suggestion
R is a popular open-source interpreted programming language used for statistical computing and graphics. It was created by Ross Ihaka and Robert Gentleman and version 1.0 was released in February 2000.
```

Since the target of both sentences appear related, let's keep them together rather than on separate lines.